pages/reference/hardware/versioning.md

```diff
@@ -15,37 +15,99 @@ In balenaCloud the device type is also used to define what architecture containe
 When looking at all the supported device types in balenaCloud, you will notice that there are a number of different designations. Here we will discuss a few of these designations in more detail.
 
 #### Official
-All device types that are not designated with `COMMUNITY` or `PRIVATE` are considered officially supported boards.
+All device types that are not designated with `PRIVATE` are considered officially supported boards.
 
 Official device types are:
 - Run through a full suite of testing on real hardware each time an OS version is released for the device type.
 - We endevour to ensure that all the hardware peripherals on the device type are functional and accessible. This includes things like i2c, SPI, audio, etc.
 - Continually updated to new versions of the OS, usually within 2 or 3 minor versions of meta-balena master version.
 - Official devices are recommended for production deployments.
 
-On some official device types you will see the `(BETA)` designation. This does not mean the OS itself is `beta` level software, but rather that this device type not yet used by enough people or has not been in production for a long enough time. In balenaCloud all official device types start out as `BETA` and when the team decides the device type has enough production usage, it is graduated out of `BETA`.
 
 #### Private
 Private device types have exactly the same guarantees and testing process as official device types but they are only visible to specific customers that have paid for this device support. If you are interested in finding out more about private device support email solutions@balena.io .
 
-#### Community
```

Comment:
Taking the opportunity to remove the concept of community supported boards. The API removed it a while ago, so this is just confusing